lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -372,6 +388,17 @@ def _user_certificate_info(self, user, context, course_grade, bulk_certs):
         )
         return certificate_info
 
+    def _enrollment_status(self, user, context):
+        """
+        Returns a list of a string that indicates whether a user
```

Comment:
Nit that you definitely don't need to address unless you want to and are already changing something else: this comment reads a bit strangely. Maybe just change "whether a user" to "whether each user"?